code/modules/power/supermatter/supermatter.dm

```diff
@@ -266,6 +266,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/cascade_initiated = FALSE
 	///Reference to the warp effect
 	var/atom/movable/supermatter_warp_effect/warp
+	///How much power the supermatter is losing.
+	var/power_loss = 0
```

Comment:
why does this need to be a var on the object?

Reply:
I kinda forgot why. Maybe for vv, or to allow someone to use it such as for NT CIMS without having to add it.